dnf-behave-tests/features/steps/repo.py

```diff
@@ -217,6 +217,11 @@ def step_set_up_metalink_for_repository(context, repo):
     create_repo_conf(context, repo)
 
 
+@behave.step("I start capturing outbound HTTP requests")
+def step_start_http_capture(context):
+    context.httpd.configure('logging', True)
```

Comment:
The patch looks good, just FTR, is it a performance concern that we don't just log by default and save ourselves the trouble to enable it explicitly?

Reply:
You're actually right, we could probably just enable it straight away. My concern back then was that the in-memory log could get unnecessarily big if there happens to be a scenario that produces a lot of requests, but now I realize it just doesn't really matter since the logs are always cleared together with the httpd instance.  So yeah, I'll probably just drop this step and enable logging by default.

Reply:
I changed my mind - I think we should keep this step so that it's clear when exactly the logging starts. The reason is, there might be other DNF operations that reach out to the repository as part of the scenario setup which we don't want to pollute our log with.